### ISSUE TITLE
Would you please provide a series of training checkpoints of Pythia-410M

### ISSUE THREAD

**Issue by Cbtor** [Gaffey/ExCP#1]
Hi, I read the paper and find this work interesting. Could you please provide a series of training checkpoints for us to experience this method quickly?

**Comment by Gaffey** [comment]
Sorry, we do not have the plan now. But you can easily reproduce the results with the provided training code.

**Comment by Cbtor** [comment]
Okay, I'm trying to train. If I get some results, I'll update here.

**Comment by Cbtor** [comment]
Hi, I make it to reproduce your experimental results with comparable accuracy and slightly smaller size (average 55MB). I upload the model to https://huggingface.co/Cbtor/ExCP, may you add it to this repo?

**Comment by Gaffey** [comment]
Thanks for your contribution! I would add the link to README.
